libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/PolicyManager.java

```diff
@@ -77,13 +82,18 @@ private static Set<Module> findSystemModules() {
             .collect(Collectors.toUnmodifiableSet());
     }
 
-    public PolicyManager(Policy defaultPolicy, Map<String, Policy> pluginPolicies, Function<Class<?>, String> pluginResolver) {
-        this.serverEntitlements = buildScopeEntitlementsMap(Objects.requireNonNull(defaultPolicy));
-        this.pluginsEntitlements = Objects.requireNonNull(pluginPolicies)
-            .entrySet()
+    public PolicyManager(
+        Policy defaultPolicy,
+        Map<String, Policy> pluginPolicies,
+        Function<Class<?>, String> pluginResolver,
+        Module entitlementsRuntimeModule
```

Comment:
nit: we don't have the term "runtime" anywhere anymore (at least not in the jar files, it's still in the package names but I plan to simplify that later), can this just be `entitlementsModule`?

Reply:
Entitlements has several modules. IMHO the bug here is removing the term "runtime" from the code. We now have this one nameless module (not to be confused with an unnamed module) and a couple other modules that have names.

However, given the current layout of the code, I agree this needs changing.